### ISSUE TITLE
[Feature Request]: Unassign a specific person accessory is added back to stock and sometimes we do not need to return to stock

### ISSUE THREAD

**Issue by Gonchitec** [snipe/snipe-it#12997]
### Is your feature request related to a problem? Please describe.

The problem is that we unassign a specific person accessory is added back to stock and sometimes we do not need to return to stock because that accessory has broken, so we need some option / category / label where to store the accessories, because we do not find the way to do it, but with assets if we can do it.

for example: we have 40 people, each of them has a mouse and a keyboard, there are times that the mouse would have to change it for another in stock, in that case no problem, but there are times that the mouse breaks, so when unassigning it from the person, snipe-it adds it back to stock).

### Describe the solution you'd like

In short, we need to have a count of accessories and components that break, as accessories are not treated the same as assets.

### Describe alternatives you've considered

Put a button in the unassign section (accessory, components) where you indicate if the accessory is broken, and in that case do not add it back to stock, but add it to a section called broken or disposable.

### Additional context

we have problems when it comes to automatically deduct the stock, since we have 200 keyboards alike and we think it should go in the accessories section, but however in that section we treat the keyboards as a single object, so we can not send a single keyboard to a scrap or broken section.

**Comment by alishk7193** [comment]
Can a Scrap option be added to the assigned items as well? usually required when we need to scrap an accessories but there is no such option. the only option is to check back in and then reduce the quantity of purchased items in accessories entry.
